### PR TITLE
updating pk-sim

### PR DESCRIPTION
**Pull request by georgeDaskalakis**

### CODE REVIEW

**Comment by msevestre** [comment]
>:\projects\pk-sim-g4na9\src\PKSim.Infrastructure\Services\ImportObservedDataTask.cs(29,67): error CS0535: 'ImportObservedDataTask' does not implement interface member 'IObservedDataConfiguration.MolWeightEditable' [C:\projects\pk-sim-g4na9\src\PKSim.Infra

still failing

**Comment by georgeDaskalakis** [comment]
> > :\projects\pk-sim-g4na9\src\PKSim.Infrastructure\Services\ImportObservedDataTask.cs(29,67): error CS0535: 'ImportObservedDataTask' does not implement interface member 'IObservedDataConfiguration.MolWeightEditable' [C:\projects\pk-sim-g4na9\src\PKSim.Infra
> 
> still failing

I know that's why I converted to a draft..weird though cause the rename of this interface member is part of the PR... i ll look into this a bit more

**Comment by msevestre** [comment]
oh sorry did not see. I got a notification that my review was required.

**Comment by georgeDaskalakis** [comment]
@msevestre after fixing the duplicate packages thing, this is ready for reveiew :)

File: src/PKSim.Assets.Images/PKSim.Assets.Images.csproj [hunk 1]
```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="11.0.72" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="11.0.72" />
+	  <PackageReference Include="OSPSuite.Assets" Version="11.0.77" />
```
Comment (msevestre) [inline src/PKSim.Assets.Images/PKSim.Assets.Images.csproj:29]: there is some weird indent in every file @georgeDaskalakis

Comment (msevestre) [reply src/PKSim.Assets.Images/PKSim.Assets.Images.csproj:29]: correction in some files. Can you clean that up?

File: src/PKSim.Assets/PKSim.Assets.csproj [hunk 2]
```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="11.0.72" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="11.0.72" />
+	  <PackageReference Include="OSPSuite.Assets" Version="11.0.77" />
```
Comment (msevestre) [inline src/PKSim.Assets/PKSim.Assets.csproj:29]: and here

File: src/PKSim.Assets.Images/PKSim.Assets.Images.csproj (hunk 1 above)
Comment (georgeDaskalakis) [reply src/PKSim.Assets.Images/PKSim.Assets.Images.csproj:29]: done

File: src/PKSim.Assets/PKSim.Assets.csproj (hunk 2 above)
Comment (georgeDaskalakis) [reply src/PKSim.Assets/PKSim.Assets.csproj]: done

**Pull request merged**